mastodon-bot.cljs

```diff
@@ -119,7 +119,7 @@
      (post-status status-text (not-empty ids)))))
 
 (defn get-mastodon-timeline [callback]
-  (.then (.get mastodon-client "timelines/home" #js {})
+  (.then (.get mastodon-client (string/join "" ["accounts/" (:account-id mastodon-config) "/statuses"]) #js {})
```

Comment:
as note you can just do `(str "accounts/" (:account-id mastodon-config) "/statuses")` here